examples/digital_fingerprinting/production/morpheus/dfp/stages/dfp_training.py

```diff
@@ -31,7 +32,7 @@
 
 class DFPTraining(SinglePortStage):
 
-    def __init__(self, c: Config, model_kwargs: dict = None):
+    def __init__(self, c: Config, model_kwargs: dict = None, epochs=30, validation_size=0.0):
```

Comment:
Need a docstring here for the new options.

Reply:
Make sure to add the range of `validation_size` to the docstring. Ask @dagardner-nv if you havent done this before.

Reply:
Issue #609 created to add docstring to all production DFP code. This docstring will be added as part of a separate PR.